src/dos/dos_memory.cpp

```diff
@@ -468,6 +469,14 @@ bool DOS_LinkUMBsToMemChain(uint16_t linkstate) {
 	return true;
 }
 
+static constexpr uint16_t kilobytes_to_segment(size_t address)
+{
+	constexpr auto bytes_per_segment = 16;
+	constexpr auto bytes_per_killobyte = 1024;
+	assert(address < 64 * bytes_per_killobyte);
+	assert(address % bytes_per_segment == 0);
```

Comment:
These two asserts seem to be some leftover from an earlier implementation. They're not needed and don't make any sense.

Reply:
Yep, I changed what I was doing here and forgot to remove these.  Gone now.